app/javascript/angular/code/controllers/_fixed_sessions_map_ctrl.js

```diff
@@ -133,7 +133,7 @@ export const FixedSessionsMapCtrl = (
       const onTimeRangeChanged = (timeFrom, timeTo) => {
         elmApp.ports.timeRangeSelected.send({ timeFrom, timeTo });
         params.update({ data: { timeFrom, timeTo } });
-        sessions.fetch();
+        $scope.sessions.fetch();
```

Comment:
oh, so this was broken before, was it?

Reply:
I think so, but there was a watch that was fetching the sessions anyway.

Reply:
thx for finding this then :)